### PR TITLE
at-spi2-core: enable non-linux

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **at-spi2-core/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ericLemanissier** [comment]
all green in https://github.com/eirikb/proof-of-conan/actions/runs/6297646874

**Review by RubenRBS — DISMISSED** [review]
Wow, nice! Thanks :)

**Comment by ericLemanissier** [comment]
blocked by:
- https://github.com/conan-io/conan-center-index/pull/19935

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/6296777590)

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/6296777590)

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/6296777590)

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/6296777590)

**Comment by ericLemanissier** [comment]
Help me SIP gods https://github.com/ericLemanissier/proof-of-conan/actions/runs/6296777590/job/17289904484#step:8:804

```
Command line: `clang -I/Users/runner/.conan2/p/dbus4f34f5e5d3656/p/include -I/Users/runner/.conan2/p/dbus4f34f5e5d3656/p/include/dbus-1.0 -I/Users/runner/.conan2/p/dbus4f34f5e5d3656/p/lib/dbus-1.0/include -I/Users/runner/.conan2/p/expat2fe1b057146dc/p/include /Users/runner/.conan2/p/b/at-spcfa401f930b9e/b/build-release/meson-private/tmpy3ej8zhj/testfile.c -o /Users/runner/.conan2/p/b/at-spcfa401f930b9e/b/build-release/meson-private/tmpy3ej8zhj/output.exe -isysroot /Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk -arch x86_64 -O0 -Werror=implicit-function-declaration /Users/runner/.conan2/p/dbus4f34f5e5d3656/p/lib/libdbus-1.dylib -lpthread /Users/runner/.conan2/p/expat2fe1b057146dc/p/lib/libexpat.dylib` -> 0
Program stdout:


Program stderr:

dyld: Library not loaded: @rpath/libdbus-1.3.dylib
  Referenced from: /Users/runner/.conan2/p/b/at-spcfa401f930b9e/b/build-release/meson-private/tmpy3ej8zhj/output.exe
  Reason: image not found


../src/meson.build:179:23: ERROR: Could not run alignment test binary.
```

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/6296777590)

**Comment by ericLemanissier** [comment]
Let's go! Wtyt @SpaceIm ?

**Review by jwillikers — APPROVED** [review]

File: recipes/at-spi2-core/new/conanfile.py
Comment (RubenRBS) [inline recipes/at-spi2-core/new/conanfile.py:117]:
```suggestion
        meson.configure()
```

**Review by RubenRBS — CHANGES_REQUESTED** [review]
Thanks! We should remove the excepting printing for the final merge - I'll merge the commit once the CI queue clears a bit :)

File: recipes/at-spi2-core/new/conanfile.py
Comment (ericLemanissier) [reply recipes/at-spi2-core/new/conanfile.py:117]: Why should we remove it?

**Review by jwillikers — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

File: recipes/at-spi2-core/new/conanfile.py
Comment (RubenRBS) [reply recipes/at-spi2-core/new/conanfile.py]: Oh, good to know!

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**
